packages/components/inputs/select-input/README.md

```diff
@@ -83,6 +83,7 @@ export default Example;
 | `menuPortalTarget`       | `ReactSelectProps['menuPortalTarget']`                                                                |          |         | Dom element to portal the select menu to&#xA;<br>&#xA;[Props from React select was used](https://react-select.com/props)                                                                                                                                                                                                                                                                                                                       |
 | `menuPortalZIndex`       | `number`                                                                                              |          | `1`     | z-index value for the menu portal                                                                                                                                                                                                                                                                                                                                                                                                              |
 | `menuShouldBlockScroll`  | `ReactSelectProps['menuShouldBlockScroll']`                                                           |          |         | whether the menu should block scroll while open&#xA;<br>&#xA;[Props from React select was used](https://react-select.com/props)                                                                                                                                                                                                                                                                                                                |
+| `closeMenuOnSelect`      | `boolean`                                                                                             |          | `true`  | whether the menu should be closed on option select&#xA;<br>&#xA;[Props from React select was used](https://react-select.com/props)                                                                                                                                                                                                                                          |
```

Comment:
Did you edit this file by hand?
There is this `generate-readmes` yarn command that handles the generation of this files automatically so its easier to handle and descriptions are the same in code comments and in these files.